code/__defines/tgs.config.dm

```diff
@@ -0,0 +1,12 @@
+#define TGS_EXTERNAL_CONFIGURATION
+#define TGS_V3_API
+#define TGS_DEFINE_AND_SET_GLOBAL(Name, Value) GLOBAL_VAR_INIT(##Name, ##Value); GLOBAL_PROTECT(##Name)
+#define TGS_READ_GLOBAL(Name) GLOB.##Name
+#define TGS_WRITE_GLOBAL(Name, Value) GLOB.##Name = ##Value
+#define TGS_WORLD_ANNOUNCE(message) to_chat(world, "<span class='boldannounce'>[html_encode(##message)]</span>")
+#define TGS_INFO_LOG(message) log_to_dd("TGS Info: [##message]")

```

Comment:
Note; this uses log_to_dd over log_world because of this BYOND bug: http://www.byond.com/forum/post/2061007

Reply:
Ye, I encountered that bug in another thing